test/e2e/scenarios/Snapshot/snapshot_update_with_tags.yaml

```diff
@@ -56,7 +38,7 @@ steps:
         conditions:
           ACK.ResourceSynced:
             status: "True"
-            timeout: 100
+            timeout: 180
```

Comment:
This can be 5 seconds and it should still work. Every tag operation can be 5 seconds IMO

Reply:
Will create a new PR to fix all timeout.

Reply:
Set it back to 100. Will find a more appropriate value in a new PR.